config/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/resource/OAuth2ResourceServerConfigurer.java

```diff
@@ -189,6 +194,13 @@ public OAuth2ResourceServerConfigurer(ApplicationContext context) {
 		return this;
 	}
 
+	public OAuth2ResourceServerConfigurer<H> bearerTokenAuthenticationConverter(
```

Comment:
I think this should take an `AuthenticationConverter` so that applications can switch out the implementation